install-components.md

```diff
@@ -160,10 +160,28 @@ that you plan to create the `Workload` in:
     ```
 
 3. Give developers namespace-level access and view access to appropriate cluster-level resources by doing one of the following:
-  * Use the `tanzu auth` plug-in to grant `app-viewer` or `app-editor` roles
-  * Apply the following RBAC policy:
+
+  **Note:** In order to apply the bindings below, an adminstrator account is required. However to use the pre-defined roles, a non adminstrator account (i.e. developer account) is required.
+
+  1) Use the [`tanzu rbac`](authn-authz/binding.md) plug-in to grant `app-viewer` or `app-editor` roles to an identity provider group
 
       ```console
+      tanzu rbac binding add -g <GROUP-FOR-APP-VIEWER> -n <YOUR-NAMESPACE> -r app-viewer
+      tanzu rbac binding add -g <GROUP-FOR-APP-EDITOR> -n <YOUR-NAMESPACE> -r app-editor
+      ```
+
+      Where:
+
+      - `<YOUR-NAMESPACE>` is the name that you want to use for the developer namespace
+      - `<GROUP-FOR-APP-VIEWER>` is the group from the upstream identity provider should have access to `app-viewer` resources on the current cluster
```

Comment:
I think this sentence could be a little confusing. Maybe something like `<GROUP-FOR-APP-VIEWER> is the object id of the group from the upstream identity provider that would like to have access to `app-viewer` resources on the current cluster` would be a bit clearer? Same for the line below as well

Reply:
I excluded group id, just because it may not be an id. It really depends on the idp